src/ember-app.js

```diff
@@ -323,6 +336,7 @@ class EmberApp {
 
     try {
       await this._visit(path, fastbootInfo, bootOptions, result, buildSandboxPerVisit);
+      result.analytics.fastbootVisitTime = Date.now() - fastbootVisitStartTime;
```

Comment:
I think `visit` end time is when `finally` block is called in order capture the entire time. Is there specific reason why we are not capturing the remaining time to build the result and shoebox?